modules/core/shared/src/test/scala/gem/util/LocationSpec.scala

```diff
@@ -9,7 +9,7 @@ import cats.kernel.laws.discipline._
 import cats.kernel.Comparison.{ GreaterThan => GT, LessThan => LT, EqualTo => EQ }
 import cats.tests.CatsSuite
 
-@SuppressWarnings(Array("org.wartremover.warts.Equals", "org.wartremover.warts.NonUnitStatements"))
+@SuppressWarnings(Array("org.wartremover.warts.Equals", "org.wartremover.warts.NonUnitStatements", "org.wartremover.warts.TraversableOps"))
```

Comment:
This test uses `.min` and `.max` which are unsafe in general but ok here so I just disabled the [new] warning.